src/App.js

```diff
@@ -192,14 +195,21 @@ export default function App() {
         />
 
         <div className="right-center-container">
-          {getTasks().map((task) => (
-            <ReadBook
-              key={task.id}
-              task={task}
-              onChange={onChange}
-              isActive={active === task.parentId}
-            />
-          ))}
+          {active === undefined ? (
```

Comment:
предлагаю сделать так:

```suggestion
          {!active ? (
```

Reply:
учту